File: 39 -  Count the Number of Duplicate Elements in an Array/solution.js
function countDuplicates(nums) {
  // Write your code here
  let freq = {};
  let duplicatedCount = 0;

  for (let num of nums) {
    freq[num] = (freq[num] || 0) + 1;
  }

  for (let [_, value] of Object.entries(freq)) {
    if (value > 1) duplicatedCount++;
  }

  return duplicatedCount;
}

// Test Cases
console.log(countDuplicates([1, 2, 3, 4, 5, 1, 2, 6, 6]));

File: 39 -  Count the Number of Duplicate Elements in an Array/solution.py
def count_duplicates(nums):
    # Write your code here
    freq = {}
    duplicated_count = 0

    for num in nums:
        freq[num] = freq.get(num, 0) + 1

    for _, value in freq.items():
        if value > 1:
            duplicated_count += 1

    return duplicated_count


# Test Cases
print(count_duplicates([1, 2, 3, 4, 5, 1, 2, 6, 6]))